src/composables/tippingContracts.ts

```diff
@@ -0,0 +1,75 @@
+import { computed, ref } from '@vue/composition-api';
+import { Contract } from '@aeternity/aepp-sdk-13';
+
+import TippingV1ACI from '../lib/contracts/TippingV1ACI.json';
+import TippingV2ACI from '../lib/contracts/TippingV2ACI.json';
+import { RUNNING_IN_TESTS } from '../lib/environment';
+import { useSdk13 } from './sdk13';
+import { watchUntilTruthy } from '../popup/utils';
+import {
+  IDefaultComposableOptions,
+  INetwork,
+  TippingV1ContractApi,
+  TippingV2ContractApi,
+} from '../types';
+
+interface TippingContracts {
+  tippingV1: Contract<TippingV1ContractApi>;
+  tippingV2?: Contract<TippingV2ContractApi>;
+}
+
+let tippingV1: Contract<TippingV1ContractApi> | undefined;
+let tippingV2: Contract<TippingV2ContractApi> | undefined;
+const initializing = ref(false);
+
+export function useTippingContracts({ store }: IDefaultComposableOptions) {
+  const { getSdk } = useSdk13({ store });
+
+  const activeNetwork = computed<INetwork>(() => store.getters.activeNetwork);
+  const tippingSupported = computed(() => store.getters.tippingSupported);
+
+  async function initTippingContracts() {
+    if (!tippingSupported && !RUNNING_IN_TESTS) {
```

Comment:
Instead of creating composable for the `tippingSupported` I suggest to use the getter directly here. This way each time we use this composable we would not need to create the instance of the computed property.